docs/adapters.md

```diff
@@ -2,6 +2,43 @@
 title: "Playback adapters"
 ---
 
-# Playback adapters!
+# Playback adapters
 
-TO BE DONE
+## Basic concept
+
+There are lot of varios formats and concept of delivering video to end user.
+Some times you could be satisfied with [progressive download](https://en.wikipedia.org/wiki/Progressive_download) of your mp4 video.
+Main pros of that approach - crossbrowser support and almost no specific requirements for your backend.
+But when user experience is really important for you, you should conside to use [adaptive streaming](https://en.wikipedia.org/wiki/Adaptive_bitrate_streaming).
+Main pros - better user experience. Main cons - there is no crossbrowser support of main formats (HLS and MPEG-DASH) of adaptive streaming. But there is various libraries that add this support to browser. But they are heavyweigth, so we are adding them to our default bundle, but we give you ability to add them to Playable.
+
+In Playable we have concept of adapters. When you have some library that is responsible for playing video of some format, that is not nativly supported in browser, you can create ES6 class that implements proper interface and you can use it inside Playable. We have couple adapters writed by ourselfs, you can use them or you can just check them for inspiration.
+
+There is 2 ways of adding them.
```

Comment:
Grammarly fixes:
![image](https://user-images.githubusercontent.com/2445828/36543869-b721df30-17ed-11e8-9334-4442dd16a48e.png)

fixes + attempt to improve:
"
In Playable we have _a concept_ of adapters. <del>When you have</del>[To use] some library that is responsible for playing video of some format,  <del>that is not _natively_ supported in _the browser_</del>, <del>you can</del> create _an ES6_ class that implements a _proper interface_ and <del>you can use it inside Playable</del> [add it to Playable as playback adapter]. We have couple adapters _written_ by _ourselves_ <del>, you can use them or you can just check them for inspiration</del>[. Use them or check code for inspiration].

There _are_ 2 ways of adding them.
"